src/status/utils/cache.nim

```diff
@@ -0,0 +1,17 @@
+import tables, times
+
+type ValueTime* = ref object
+  value*: string
+  timestamp*: DateTime
+
+type CachedValues* = Table[string, ValueTime]
+
+proc newCachedValues*(): CachedValues = initTable[string, ValueTime]()
```

Comment:
could the duration be a parameter instead of the hardcoded 5 minutes?

Reply:
not in this line afak, but it could a param for `isCached` with a default value